source/_integrations/http.markdown

```diff
@@ -105,6 +105,21 @@ http:
 
 The [Set up encryption using Let's Encrypt](/blog/2015/12/13/setup-encryption-using-lets-encrypt/) blog post gives you details about the encryption of your traffic using free certificates from [Let's Encrypt](https://letsencrypt.org/).
 
+<div class='note'>
+
+  When using a reverse proxy, you will need to enable the `use_x_forwarded_for` and `trusted_proxies` options. Requests from reverse proxies will be blocked after 2021.7 if these options are not set.
```

Comment:
Please remove the version reference, we don't document history, but the current state. For changelogs the release notes are available.